compiler/rustc_codegen_ssa/src/sir.rs

```diff
@@ -215,6 +227,20 @@ impl SirFuncCx<'tcx> {
         this
     }
 
+    /// Compute layout and offsets required for blackholing.
+    pub fn compute_layout_and_offsets(&mut self) {
+        let mut layout = Layout::from_size_align(0, 1).unwrap();
```

Comment:
Why `1` here? I suppose alignment doesn't matter for zero bytes?

Reply:
That's just the minimum allowed alignment: https://doc.rust-lang.org/std/alloc/struct.Layout.html#impl

Reply:
I see. Does `Layout::new::<()>()` work?

Reply:
No idea. I just followed the documentation: https://doc.rust-lang.org/std/alloc/struct.Layout.html#examples